app/lib/frontend/templates/views/pkg/package_list_experimental.mustache

```diff
@@ -33,21 +33,28 @@
     {{/is_external}}
     {{^is_external}}
     <p class="packages-metadata">
-      v <a href="{{& url}}">{{version}}</a>
-      {{#show_prerelease_version}} / <a href="{{& prerelease_version_url}}">{{prerelease_version}}</a>{{/show_prerelease_version}}
-      • Updated: <span>{{last_uploaded}}</span>
+      <span class="packages-metadata-block">
+        v <a href="{{& url}}">{{version}}</a>
+        {{#show_prerelease_version}} / <a href="{{& prerelease_version_url}}">{{prerelease_version}}</a>{{/show_prerelease_version}}
+        • Updated: <span>{{last_uploaded}}</span>
+      </span>
       {{#publisher_id}}
-      <img class="packages-vp-icon"
+      <span class="packages-metadata-block">
+        <img class="packages-vp-icon"
            src="{{& static_assets.img__verified-publisher-icon_svg}}"
            title="Published by a pub.dev verified publisher" />
-      <a href="/publishers/{{publisher_id}}">{{publisher_id}}</a>
+        <a href="/publishers/{{publisher_id}}">{{publisher_id}}</a>
+      </span>
       {{/publisher_id}}
       {{#is_flutter_favorite}}
-        <span class="package-badge">
-          <img class="package-badge-icon" src="{{static_assets.img__flutter-logo-32x32_png}}" />
-          Flutter Favorite
-        </span>
+      <span class="package-badge">
+        <img class="package-badge-icon" src="{{static_assets.img__flutter-logo-32x32_png}}" />
+        Flutter Favorite
+      </span>
       {{/is_flutter_favorite}}
+      {{#is_null_safe}}
+      <span class="package-badge">null-safe</span>
```

Comment:
In the user interface the branding is "null-safety".

So we might have to say "null-safety compatible".

@mit-mit, do we have better terminology for use on the website?

(note. this is still behind experimental, we should keep it so, even when we ship the new design, as this will launch independently).

Reply:
> (note. this is still behind experimental, we should keep it so, even when we ship the new design, as this will launch independently).

I guess we'll need another layer of experimental, otherwise this will slip.

Reply:
What is the string used for? Is this the tool-tip?

Reply:
@mit-mit: see the screenshot at the bottom of the description

Reply:
I suggest the text of the badge is 'null safety' and that the tooltip is 'Supports the null safety language feature'

Reply:
@isoos WDYT?

Reply:
I've just now updated the PR with the text and an extra flag so that we don't necessarily launch it with the rest of the new UI. I'll update the screenshot soon.

Reply:
Screenshot updated above